includes/api/class-wc-admin-rest-onboarding-levels-controller.php

```diff
@@ -0,0 +1,256 @@
+<?php
+/**
+ * REST API Onboarding Levels Controller
+ *
+ * Handles requests to /onboarding/levels
+ *
+ * @package WooCommerce Admin/API
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Onboarding Levels controller.
+ *
+ * @package WooCommerce Admin/API
+ * @extends WC_REST_Data_Controller
+ */
+class WC_Admin_REST_Onboarding_Levels_Controller extends WC_REST_Data_Controller {
+	/**
+	 * Endpoint namespace.
+	 *
+	 * @var string
+	 */
+	protected $namespace = 'wc/v4';
+
+	/**
+	 * Route base.
+	 *
+	 * @var string
+	 */
+	protected $rest_base = 'onboarding/levels';
+
+	/**
+	 * Register routes.
+	 *
+	 * @since 3.5.0
```

Comment:
Let's drop all the `@since` lines here. We can add them in when code eventually gets merged to core. (We should also clean up other lines in the repo since we don't know what version code will be in yet).

Reply:
You got it! 👌 Dropped all other `@since` tags from around the repo as well.